app/helpers/link_helper.rb

```diff
@@ -94,19 +94,19 @@ def ontology_link_to(resource)
     end
   end
 
-  def sort_link_list(collection)
+  def sort_mapping_list(collection)
     hash = {}
 
-    collection.each_with_index do |link, i|
-      if link.symbol_mappings.empty?
-        hash["empty#{i}"] = [{link: link, target: ""}]
+    collection.each_with_index do |mapping, i|
+      if mapping.symbol_mappings.empty?
+        hash["empty#{i}"] = [{mapping: mapping, target: ""}]
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.